common/src/test/java/io/novafoundation/nova/common/utils/formatting/SpannableFormatterTest.kt

```diff
@@ -0,0 +1,4 @@
+package io.novafoundation.nova.common.utils.formatting
+
+class SpannableFormatterTest {
+}
```

Comment:
Have you planned to write some tests here?

Reply:
No